l10n/de.js

```diff
@@ -11,6 +11,7 @@ OC.L10N.register(
     "Auto" : "Automatisch",
     "Name" : "Name",
     "Mail Address" : "E-Mail-Adresse",
+    "Aliases" : "Aliase",
```

Comment:
Please revert any l10n updates. These are done via Transifex and a cron job, not manually.

Reply:
Done